### PR TITLE
Fix #207: Removed Seanlip from server job admins

### LINKED ISSUE #207: Remove Sean from server job admins
Following @seanlip's comment on https://github.com/oppia/oppia/pull/11967#issuecomment-790909157, he needs to be removed from the list of server job admins. 

The list of server job admins can be found in the `userWhitelist.json` file.

### PR DESCRIPTION
**Pull request by SAEb-ai**
## Explanation
Fixed #207. This PR removes Seanlip from Server job admins

## Checklist
- [X] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [X] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

**Comment by SAEb-ai** [comment]
@jameesjohn PTAL.

File: userWhitelist.json [hunk 1]
```diff
@@ -31,5 +31,5 @@
   "oppiaMaintainers": "oppia/core-maintainers",
   "oppiaReleaseCoordinators": "oppia/release-coordinators",
   "oppiaDevWorkflowTeam": "oppia/dev-workflow-team",
-  "serverJobAdmins": ["seanlip", "vojtechjelinek"]
+  "serverJobAdmins": "vojtechjelinek"
```
Comment (vojtechjelinek) [inline userWhitelist.json:34]: Did you test this? I assume it still needs to be a list.

**Review by vojtechjelinek — CHANGES_REQUESTED** [review]
Thanks! One comment.

File: userWhitelist.json (hunk 1 above)
Comment (SAEb-ai) [reply userWhitelist.json]: Done.

**Comment by SAEb-ai** [comment]
@vojtechjelinek PTAL.
I have addressed your review comments

**Review by vojtechjelinek — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**